src/Controls/src/Core/BindablePropertyConverter.cs

```diff
@@ -23,6 +24,13 @@ public override bool CanConvertTo(ITypeDescriptorContext context, Type destinati
 
 		object IExtendedTypeConverter.ConvertFromInvariantString(string value, IServiceProvider serviceProvider)
 		{
+			if (!RuntimeFeature.IsXamlRuntimeParsingSupported)
+			{
+				throw new NotSupportedException("XAML runtime parsing is not supported. " +
```

Comment:
Should this be a localized message ?

Reply:
I don't think this should be localized but I'm not sure what our guidelines for localization are. This exception shouldn't be reached under normal circumstances and it's mostly there to guide the trimmer. Also, maybe the `NotSupportedException` isn't the right exception here and I should change it to `InvalidOperationException` (to keep it in line with https://github.com/dotnet/maui/blob/net9.0/src/Controls/src/Core/ResourceDictionary.cs#L65 ?)

Reply:
we don't localize runtime exception. only compile time ones